collection/endpoint.csv

```diff
@@ -1416,3 +1416,6 @@ eedca3f7642b7e59f8030b734b93f95d8807c4843f327471e0e3483ad7e55740,https://www.nor
 a9f60d8290d44b2e1360b177252f9092a2e08156d49012460d396ce81ab78b9b,https://www.blackpool.gov.uk/Residents/Planning-environment-and-community/Documents/2023-Brownfield-register.csv,,,2024-01-16,2024-01-12,
 86e393d39e585f3c4f3f1b13a51c3942ee311386101ab34da55df8c20d400abd,https://www.lakedistrict.gov.uk/planning/planningpolicies/brownfield-land-register2/ldnpa_brownfieldregister_2023-12-11_rev6.csv,,,2024-01-17,2024-01-17,
 0376dd77a823a598dac505102636b96c63e3edb0dba998099c2db757da16c21b,https://data.yorkopendata.org/dataset/7b937604-a03f-406b-a50c-e0ae1e260d1e/resource/80b83b2b-f964-425e-9609-5a7a8b66bab1/download/brownfieldlandregisterfinal2023-1.csv,,,2024-01-18,2024-01-17,
+1a6debeebd17e2f960508483c53ee1a86ff084a2b60fc51f5a79620a2de247bf,https://www.staffsmoorlands.gov.uk/media/4895/brownfield-land.csv-2023/Excel/sm23.brownfield-land.csv?m=1705316336557,,,2024-01-22,2024-01-15,
+7ae26c52991f86909cd53f1ec997f6e338e78b884e29778e6d6542efec1b8e25,https://www.highpeak.gov.uk/media/4893/Brownfield-Land-Register-2023-CSV/Excel/hp23.brownfield-land_register.csv?m=1703059825267,,,2024-01-22,2024-01-15,
+7d8a0ebf55ccc27484cd5341e604fc57835b3c5baec2209dec8b2f2c1d58d153,https://lewisham.gov.uk/-/media/brownfield-land-register-december-2023.ashx,,,2024-01-22,2024-01-17,

```

Comment:
Looking at the raw CSV file for Lewisham it looks like it does have some unassigned entities

There are a couple of extra lines at the start
When I removed those and ran locally (just to check) it throws an 'invalid org' error

I realised in the CSV file the OrganisationURI field has invalid data in it (ideally it should be a URL)

Maybe we get back to them on this and add the other two endpoints?